products/paas/repository.md

```diff
@@ -0,0 +1,82 @@
+# Repository
+
+The source code of your project will reside in a git-based VCS repository. You can start with a plain project, however we suggest starting with a fork of the [shopware/paas](https://github.com/shopware/paas) setup template from GitHub. You will find out more about the setup template in the [Setup Template](setup-template.md) section.
+
+## Fork and clone setup template
```

Comment:
There are also integrations like for GitLab. I guess that is very important for agencies